actors/tableland/src/lib.rs

```diff
@@ -84,6 +84,15 @@ impl Actor {
         Ok(QueryReturn { cols, rows: res })
     }
 
+    pub fn ping<RT>(rt: &RT) -> Result<PingReturn, ActorError>
```

Comment:
introduced a temporary method for this test to ensure the communication between FVM actors and evm contracts is working fine. we can remove it later with something useful.

Reply:
very cool